src/app/machines/components/MachineHeaderForms/ActionFormWrapper/ActionFormWrapper.test.tsx

```diff
@@ -16,42 +14,31 @@ import {
   rootState as rootStateFactory,
   tagState as tagStateFactory,
 } from "testing/factories";
+import { renderWithBrowserRouter } from "testing/utils";
 
 const mockStore = configureStore();
 
-it("can set selected machines to those that can perform action", async () => {
-  const state = rootStateFactory();
-  const machines = [
-    machineFactory({ system_id: "abc123", actions: [NodeActions.ABORT] }),
-    machineFactory({ system_id: "def456", actions: [] }),
-  ];
-  const store = mockStore(state);
-  render(
-    <Provider store={store}>
-      <MemoryRouter
-        initialEntries={[{ pathname: "/machines", key: "testKey" }]}
-      >
-        <CompatRouter>
-          <ActionFormWrapper
-            action={NodeActions.ABORT}
-            clearHeaderContent={jest.fn()}
-            machines={machines}
-            viewingDetails={false}
-          />
-        </CompatRouter>
-      </MemoryRouter>
-    </Provider>
-  );
+afterEach(() => {
+  jest.restoreAllMocks();
+});
 
-  await userEvent.click(
-    screen.getByRole("button", { name: /update your selection/ })
+it("scrolls to the top of the window when opening the form", async () => {
+  global.scrollTo = jest.fn();
+  // eslint-disable-next-line testing-library/no-node-access
+  const html = document.querySelector("html");
+  if (html) {
+    // Move the page down so that the hook will fire.
+    html.scrollTop = 10;
+  }
+  renderWithBrowserRouter(
+    <ActionFormWrapper
+      action={NodeActions.ABORT}
+      clearHeaderContent={jest.fn()}
+      machines={[]}
+      viewingDetails={false}
+    />
   );
-
-  const expectedAction = machineActions.setSelected(["abc123"]);
-  const actualActions = store.getActions();
-  expect(
-    actualActions.find((action) => action.type === expectedAction.type)
-  ).toStrictEqual(expectedAction);
+  expect(global.scrollTo).toHaveBeenCalled();
```

Comment:
```suggestion
  expect(global.scrollTo).toHaveBeenCalled();
  global.scrollTo = originalScrollTo;
```

Reply:
as scrollTo is not implemented in js-dom this will essentially be unsetting `global.scrollTo` but that's probably the safest thing to do to prevent test leakage. I'll do this in an `afterAll`.